pkgs/applications/video/manim-community/default.nix

```diff
@@ -0,0 +1,94 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, pythonOlder
+, cairo
+, ffmpeg_3
+, sox
+, xdg-utils
+, colour
+, numpy
+, mapbox-earcut
+, pillow
+, progressbar
+, scipy
+, tqdm
+, pycairo
+, pydub
+, fetchPypi
+, manimpango
+, pytestcov
+, poetry-core
+, texlive
+, moderngl-window
+, moderngl
+, networkx
+, pygments
+, rich
+, setuptools
+, pytestCheckHook
+}:
+
+buildPythonApplication rec {
+  pname = "manim-community";
+  version = "0.5.0";
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    pname = "manim";
+    inherit version;
+    sha256 = "e4284648cbb5d463c6dbced1d4e60fa9b68c14f4bb3b10c8eab0f8f49ca5da88";
+  };
+
+  patches = [ ./remove-dependency-constraints.patch ];
+
+  nativeBuildInputs =  [ poetry-core ];
+
+  # install_requires:
+  propagatedBuildInputs = [
+    pillow
+    colour
+    manimpango
+    mapbox-earcut
+    moderngl-window
+    moderngl
+    networkx
+    numpy
+    pycairo
+    pydub
+    pygments
+    rich
+    scipy
+    setuptools
+    tqdm
+    cairo
+    sox
+    ffmpeg_3
+    xdg-utils
+    # TODO: figure out which packages are exactly required
+    texlive.combined.scheme-basic
+  ];
+
+  doCheck = false;
+  checkInputs = [ pytestCheckHook ];
```

Comment:
Why are tests disabled?

Reply:
See comment in current version of the file